examples/ActiveSensing/ActiveSensing.ino

```diff
@@ -0,0 +1,28 @@
+#include <MIDI.h>
+
+///
+
+struct MySettings : public MIDI_NAMESPACE::DefaultSettings
+{
+    static const bool UseSenderActiveSensing = true;
+};
+
+MIDI_CREATE_CUSTOM_INSTANCE(HardwareSerial, Serial1, MIDI, MySettings);
+
+void setup()
+{
+  pinMode(LED_BUILTIN, OUTPUT);
+  MIDI.begin(1);                      // Launch MIDI and listen to channel 4
```

Comment:
The comment and code are not in sync here

Reply:
Correct @franky47, I'm also adding the exception that occurs when ActiveSensing does not receive a response within 300ms. Stay tuned as I update this PR.